scripts/reflow_comments.py

```
@@ -0,0 +1,163 @@
+                room + len(content[k]) + 1 + len(content[k + 1].split()[0])
+                <= LIMIT for k in range(len(content) - 1))
+            if not flagged:
+                continue
+            key = f"{path.replace(os.sep, '/')}:{start + 1}"
+            if any("  " in c for c in content) or key in SKIP:
+                skipped += 1
+                if not APPLY:
+                    print(f"SKIPPED {key}")
+                continue
+            refilled = [
+                f"{indent}// {l}" for l in textwrap.wrap(
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)